src/_help/getting-started/concepts.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Main concepts
+---
+
+- TOC
+{:toc}
+
+Before we proceed further, we wanted to share some vocabulary to clarify what we are talking about throughout this documentation.<br>
+The terminology used here aims to be as accurate as possible, but you may encounter different terms elsewhere. We have chosen to use the most common and meaningful terms.
+
+## API definition
+
+An API definition is the representation of an API written in compliance with an existing specification format.<br>
+It is sometimes referred to as an API schema, API contract, or even API specification.
```

Comment:
We should probably say here that referring to this as an `API specification` is a strong mistake (and point to the API specification concept).